tests/tasks/Cloudwatch/cw-metric.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name:  cloudwatch
+  namespace: tekton-pipelines
+spec:
+  description: "cw task to push one data point to AWS CloudWatch."
+  params:
+    - name: region
+      default: "us-west-2"
+    - name: desired-nodes
```

Comment:
Please have generic naming so we don't tie this task to only emit metrics based on nodes. We can abstract it out and pass meaning values through our pipelines.